Funda mapjes - kopie/styles/nieuwefundastyle.css

```
@@ -0,0 +1,342 @@
+:root{
+    --primaire-color: #F7A100;
+    --secundaire-color: #0071B3;
+    --primaire-lettertype: Proxima Nova;
+    --navbarlettertype: 15px;
+}
+
+.maandlasten {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: proxima nova, sans-serif;
+}
+
+/* BEGIN HEADER */
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    /* width: 414px; */
+    height: 60px;
+    border: 0px;
+    top: 0px;    
+    background-color: var(--primaire-color);
+}
+
+.navbar-left {
+    display: flex;
+    justify-content: space-between;
+}
+
+.button-menu {
+    height: 100% ;
+    background-color: var(--primaire-color);
+    border-color: var(--primaire-color);
+    border: 0px;
+    color: white;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 12px;
+    margin-left: 15px;
+    width: 50px;
+}
+
+.logo{
+    position: relative;
+    width: 100px;
+    height: 75px;
+}
+
+.button-house {
+    display: flex;
+    width: 70px;
+    height: 100%;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-color: var(--primaire-color);
+    border: 0px;
+    color: white;
+    text-decoration: none;
+    font-size: 12px;
+}
+
+.my-house{
+    display: flex;
+}
+
+.alog{
+    display: flex;
+    width: 50px;
+    height: 100%;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-color: var(--primaire-color);
+    border: 0px;
+    color: white;
+    text-decoration: none;
+    font-size: 12px;
+    margin-right: 20px;
+}
+
+
+
+
+/* -------------------------------------------- */
+
+.container {
+    max-width: 1200px;
+    width: 100%;
+}
+.image-listo{
+    display: none;
+}
+.slider-wrapper .slide-button {
+    position: relative;
+}
+
+.slider-wrapper .slide-button {
+    position: absolute;
+    height: 25px;
+    width: 25px;
+    color: #fff;
+    border: none;
+    outline: none;
+    background: var(--secundaire-color);
+    cursor: pointer;
+    border-radius: 50%;
+    top: 25%; 
+   transform: translateY(-50%);
+   display: block;
+}
+
+.slide-wrapper .slide-button:hover {
+    background:  #444;
+}
+
+.slider-wrapper .slide-button#prev-slide {
+    left: -1px;
+} 
+
+.slider-wrapper .slide-button#next-slide {
+    right: -1px;
+} 
+
+.slide-button#prev-slide {
+    max-width: 1200px;
+}
+
+.slide-button#next-slide {
+    max-width: 1200px;
+}
```

Dit zou je ook op een eenvoudigere manier kunnen schrijven door het te hergebruiken. 
```suggestion
.slide-button {
    max-width: 1200px;
}
```